### ISSUE TITLE
missing asa-ctrl

### ISSUE THREAD

**Issue by nogasgofast** [mschnitzer/ark-survival-ascended-linux-container-image#57]
I was looking around the containers file system for asa-ctrl but I can't seem to find it:
asa-server-1:/etc # whereis asa-ctrl
asa-ctrl:
just coming up blank even though I logged in as root. Tried without root as well. 

The goal was to use this:
```python
    healthcheck:
      test: asa-ctrl rcon --exec "listplayers"
      interval: 60s
      timeout: 2s
      retries: 5
      start_period: 240s
``` 
This way I could shut everything down, and start just one instance to do any updates that need doing. That's not important though I was just real confused when I could not find asa-ctrl.

**Comment by mschnitzer** [comment]
You are right, `whereis` returns nothing for me either, strange... However, the `asa-ctrl` executable is located at `/usr/bin`.

**Comment by nogasgofast** [comment]
That is where I thought it would be.
```
:/etc/docker/compose/asa-maps# docker ps
CONTAINER ID   IMAGE                                COMMAND                  CREATED         STATUS                           PORTS                                                        NAMES
34f3e1a1bea7   mschnitzer/asa-linux-server:latest   "/usr/bin/start_serv…"   3 minutes ago   Restarting (127) 5 seconds ago                                                                asa-server-2
6398e866a59b   mschnitzer/asa-linux-server:latest   "/usr/bin/start_serv…"   3 minutes ago   Up 3 minutes                     0.0.0.0:7777-7778->7777-7778/udp, 0.0.0.0:27020->27020/tcp   asa-server-1
root@backupServer:/etc/docker/compose/asa-maps# docker exec -it asa-server-1 bash
gameserver@asa-server-1:~> ls /usr/bin/asa-ctrl
ls: cannot access '/usr/bin/asa-ctrl': No such file or directory

/etc/docker/compose/asa-maps# docker exec -it -u root asa-server-1 bash
asa-server-1:/home/gameserver # ls /usr/bin/asa-ctrl
ls: cannot access '/usr/bin/asa-ctrl': No such file or directory
```

**Comment by nogasgofast** [comment]
Of note, I did just destroy these volumes and re-build them. And it was still missing. So it seems like it's not in the image or possibly not building correctly... But i didn't really mess with the volumes that much. Everything is accessible and writeable. Or should be.

**Comment by nogasgofast** [comment]
docker image inspect mschnitzer/asa-linux-server
[
    {
        "Id": "sha256:4e871afc8e1673ec455e47f821b4b6d3a35bad184dacce5cf510ae8aa7bffff1",
        "RepoTags": [
            "mschnitzer/asa-linux-server:latest"
        ],
        "RepoDigests": [
            "mschnitzer/asa-linux-server@sha256:d72ef9080bb9a8ac180e84deffaf588f43bbc7e716f8173bc1a7a567380f0273"
        ],
        "Parent": "",
        "Comment": "",
        "Created": "2023-11-04T23:58:46Z",
        "DockerVersion": "",
        "Author": "Manuel Schnitzer <github@mschnitzer.de>",
        "Config": {
            "Hostname": "",
            "Domainname": "",
            "User": "",
            "AttachStdin": false,
            "AttachStdout": false,
            "AttachStderr": false,
            "Tty": false,
            "OpenStdin": false,
            "StdinOnce": false,
            "Env": null,
            "Cmd": [
                "/bin/bash"
            ],
            "Image": "",
            "Volumes": null,
            "WorkingDir": "/home/gameserver",
            "Entrypoint": null,
            "OnBuild": null,
            "Labels": null
        },
        "Architecture": "amd64",
        "Os": "linux",
        "Size": 676347328,
        "GraphDriver": {
            "Data": {
                "MergedDir": "/var/lib/docker/overlay2/8e6917d3b00f932df6f3fee4249d3cd358ebd44e27e98e3ae0dd2c10331d6a6f/merged",
                "UpperDir": "/var/lib/docker/overlay2/8e6917d3b00f932df6f3fee4249d3cd358ebd44e27e98e3ae0dd2c10331d6a6f/diff",
                "WorkDir": "/var/lib/docker/overlay2/8e6917d3b00f932df6f3fee4249d3cd358ebd44e27e98e3ae0dd2c10331d6a6f/work"
            },
            "Name": "overlay2"
        },
        "RootFS": {
            "Type": "layers",
            "Layers": [
                "sha256:b01209261a1dc28be67ab483230348f829146e0a32e27f721b4f4c58ae5b5897"
            ]
        },
        "Metadata": {
            "LastTagTime": "0001-01-01T00:00:00Z"
        }
    }
]

**Comment by mschnitzer** [comment]
`"Created": "2023-11-04T23:58:46Z",`

You are using an outdated version of the image which doesn't contain `asa-ctrl`.

You can run `docker pull mschnitzer/asa-linux-server` to update to version 1.3.0.

Keep in mind that you need to adjust the `docker-compose.yml` file a bit, so that it has the same structure like this one: https://github.com/mschnitzer/ark-survival-ascended-linux-container-image/blob/main/docker-compose.yml

Before you do all of that, please make a copy of your server files first. In theory your docker volumes containing your savegames won't be touched, but better be safe than sorry.

A copy can be made by accessing `/var/lib/docker/volumes/` and then copy the `asa-linux-server_server-files-1` directory to a save place (make sure the server is stopped!) outside of this directory.
